lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -216,6 +219,13 @@ Below are some of the requirements and limitations of the application while runn
    * The LwM2M carrier library is currently only certified for the *LTE-M* LTE mode.
    * The :option:`CONFIG_LTE_NETWORK_USE_FALLBACK` should be disabled in your application, as seen in the :ref:`lwm2m_carrier` sample project configuration (:file:`nrf/samples/nrf9160/lwm2m_carrier/prj.conf`).
 
+* The LwM2M carrier library registers to receive several AT event reports using the :ref:`at_cmd_readme` and :ref:`at_notif_readme` libraries. The following notifications should not be deregistered by the application:
+
+   * SMS events (AT+CNMI).
+   * Packet Domain events (AT+CGEREP).
+   * Report Network Error Codes events (AT+CNEC): EPS Session Management events are used by the LwM2M carrier library. The application may enable or disable EPS Mobility Management events.
+   * Network Registration Status events (AT+CEREG): Notification Level 2 is used by the LwM2M carrier library. The application may increase this level, but should not decrease it.
+
```

Comment:
Do we rely on `AT+CMEE` to return the correct error codes?

Reply:
No we don't.